scalableminds/webknossos
# also clear new datasets cache key for global localStorage key

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- it's a bit hard to test (which is why this slipped in earlier), which is why I would suggest to have a hard look at the one line change. inspecting my current localStorage for the mh wk instance, I see the following keys:
`email@address-orgaName-wk.datasets-v2`
`email@address-wk.datasets-v2`
- the second one is deprecated (doesn't use the orga name), but uses the new dataset key (since the version didn't need to be bumped).
- the line change should delete this entry.

### Issues:
- helps for #5038 
------
